include/powsybl/iidm/Network.hpp

```diff
@@ -66,10 +72,14 @@ class Network : public Container, public VariantManagerHolder {
 
     static Network readXml(std::istream& istream, const converter::ImportOptions& options, const converter::Anonymizer& anonymizer);
 
+    static Network readXml(std::istream& istream, const stdcxx::Properties& parameters, const converter::Anonymizer& anonymizer);
```

Comment:
It could be nice to propose two overloads without the anonymizer, or to add a constructor to ImportOptions and ExportOptions that takes a Properties object.

The idea is to keep the API user-friendly

Reply:
I agree with creating constructors which take a `stdcxx::Properties` as parameter, but it adds a dependency between `converter` and `converter::xml` module. Can we accept it (related to https://github.com/powsybl/powsybl-iidm4cpp/pull/106#discussion_r435832078)?